source/configuration_manual/forwarding_parameters.rst

```diff
@@ -6,80 +6,140 @@ Forwarding parameters in IMAP/POP3/LMTP/SMTP proxying
+``ADDR``     Client IP
+``PORT``     Client port
+``SESSION``  Session ID
+``TTL``      TTL which is reduced by each hop, loop prevention. When TTL drops
+             to 0, the connection is dropped.
+``FORWARD``  Base64-encoded, tab-separated list of ``key=value`` pairs to be
+             forwarded to auth process. The keys and values are escaped using
+             Dovecot's tab-escape format.
+============ ===================================================================
 
 SMTP/LMTP protocol
 ------------------
 
 See http://www.postfix.org/XCLIENT_README.html
 
--  ``ADDR`` - Client IP, IPV6: prefix is needed for IPv6.
-
--  ``PORT`` - Client port
-
--  ``SESSION`` - Session ID
-
--  ``HELO`` - Original HELO/EHLO
-
--  ``LOGIN`` - Original LOGIN value
-
--  ``TIMEOUT`` - Original TIMEOUT
-
--  ``TTL`` - TTL which is reduced by each hop, loop prevention. When TTL
-   drops to 0, the connection is dropped.
-
--  ``PROTO`` - Forwarded protocol, can be one of SMTP, ESTMP, LMTP.
-
--  ``FORWARD`` - Base64 encoded key=value parameter to be stored.
+Supported parameters (SMTP & LMTP)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+============ ===================================================================
+Parameter    Description
+============ ===================================================================
+``ADDR``     Client IP; prefix (``IPV6:<ipv6_address>``) is required for IPv6.
+             However, Dovecot currently forwards without the IPv6 prefix,
+             which does not follow the correct Postfix XCLIENT syntax.
+``PORT``     Client port
+``TTL``      TTL which is reduced by each hop, loop prevention. When TTL drops
+             to 0, the connection is dropped.
+``HELO``     Original ``HELO``/``EHLO``
+``LOGIN``    Original ``LOGIN`` value
+``TIMEOUT``  Original ``TIMEOUT``
+``PROTO``    Forwarded protocol: ``SMTP``, ``ESTMP``, or ``LMTP``.
+============ ===================================================================
+
+Supported parameters (*SMTP/Submission* ONLY)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+============ ===================================================================
+Parameter    Description
+============ ===================================================================
+``FORWARD``  Base64-encoded, tab-separated list of ``key=value`` pairs to be
+             forwarded to auth process. The keys and values are escaped using
+             Dovecot's tab-escape format. This value is effectively limited
+             to around 1000 bytes.
+``SESSION``  Session ID
+============ ===================================================================
+
+Supported parameters (*LMTP* ONLY)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+================= ==============================================================
+Parameter         Description
+================= ==============================================================
+``XRCPTFORWARD``  Base64-encoded, tab-separated list of ``key=value`` pairs to
```

Comment:
Should clarify that this is not XCLIENT parameter. This is RCPT TO command parameter.